mobx/lib/src/core/context.dart

```diff
@@ -6,7 +6,7 @@ class _ReactiveState {
   int nextIdCounter = 0;
 
   Derivation trackingDerivation;
-  List<ReactionImpl> pendingReactions = [];
+  List<Reaction> pendingReactions = [];
```

Comment:
does pendingReactions need to be public?

Reply:
Its part of the `ReactiveState`, which is already private. So technically the whole state is private to the context.

Reply:
merging for now